### ISSUE TITLE
Implementing with GANs, loss argument

### ISSUE THREAD

**Issue by terrainer** [andrewjc/PyTorch-AdamL#2]
Hi,

Thank you for your code. I am trying to implement and test this with [neosr](https://github.com/muslll/neosr).

In the case of a discriminator network where the loss outputs are l_d_real and l_d_fake, what should one pass to AdamL as the loss argument?

As I understand from the paper, it should be `loss = l_d_real - l_d_fake`.

Kind regards,
terrainer
